modules/core/client/scss/components/forms.scss

```diff
@@ -258,6 +258,26 @@ $date-time-picker-btn-hover-bg: #ddd;
 	}
 }
 
+// Legacy Datepicker
+.date-picker {
```

Comment:
Does moving the CSS class here mean all components that have/will have a date picker will inherit it, rather than having it specified for each component (e.g.: the _project-comment_ as below)?